packages/remote-config/android/src/main/java/io/invertase/firebase/config/UniversalFirebaseConfigModule.java

```diff
@@ -109,12 +108,15 @@
       }
 
       if (xmlResourceParser != null) {
-        FirebaseRemoteConfig.getInstance(firebaseApp).setDefaults(resourceId);
-        return null;
+        // REVIEW this changed from setDefaults to setDefaultsAsync returning Task<Void> -
```

Comment:
@Salakar this change is functional, but grosses me out for the reason stated in the comment. Any wise user of the Task APIs could probably fix this to be non-gross in a heartbeat since the API we forward-ported to *returns* the thing our wrapper API is supposed to return